### PR TITLE
Style overhaul

### PR DESCRIPTION
**Pull request by aleangel1212**

### CODE REVIEW

File: blockonomics-woocommerce.php [hunk 1]
```diff
@@ -40,29 +40,32 @@
...
+require_once ABSPATH . 'wp-admin/includes/plugin.php';
 
-if ( is_plugin_active( 'woocommerce/woocommerce.php') || class_exists( 'WooCommerce' )) {
+if (is_plugin_active('woocommerce/woocommerce.php') || class_exists('WooCommerce')) {
     function blockonomics_woocommerce_init()
     {
         if (!class_exists('WC_Payment_Gateway')) {
             return;
         }
+
```
Comment (shivaenigma) [inline blockonomics-woocommerce.php]: Is this the file where user is suppose to add his custom css ? We need to create this file with comments as shown here https://www.elegantthemes.com/blog/tips-tricks/how-to-safely-add-custom-css-styles-to-plugins

**Comment by shivaenigma** [comment]
Styles are not same as previous. Here is what I got after merge
screenshot from 2017-10-27 12-15-15
screenshot from 2017-10-27 12-15-48

**Comment by shivaenigma** [comment]
We may also want to enque another css file (having only doc strings) where customer can make changes https://www.elegantthemes.com/blog/tips-tricks/how-to-safely-add-custom-css-styles-to-plugins

**Comment by aleangel1212** [comment]
I made the modifications we talked about and should be ready for merge

**Comment by blockonomics** [comment]
fixes #18

**Pull request merged**